### PR TITLE
T12

### PR DESCRIPTION
**Pull request by Iris314**

### CODE REVIEW

File: controllers/games.controller.js [hunk 1]
```diff
@@ -71,3 +72,12 @@ exports.postReviewCommentById = (req, res, next) => {
       } else next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const commentId = req.params.comment_id;
+  removeCommentById(commentId)
+    .then(() => {
+      res.status(200).send();
```
Comment (k-huggs) [inline controllers/games.controller.js:80]: have a look at the status code for a delete

File: __tests__/app.test.js [hunk 2]
```diff
@@ -422,29 +422,3 @@ describe("/api/reviews/:review_id/comments", () => {
     });
   });
 });
```
Comment (k-huggs) [inline __tests__/app.test.js:424]: There's no test for the delete end point

**Comment by k-huggs** [comment]
Hi, pls add some tests for the delete end point

File: __tests__/app.test.js (hunk 2 above)
Comment (Iris314) [reply __tests__/app.test.js:424]: Seems like my tests got lost during the merge. I will rewrite them and submit the PR again when finished.

**Comment by corkesw** [comment]
Good to merge

**Pull request merged**